traffic_ops/app/db/seeds.sql

```diff
@@ -97,6 +144,23 @@ FROM public.role
 WHERE "name" in ('operations', 'read-only', 'portal', 'federation', 'steering')
 ON CONFLICT DO NOTHING;
 
+INSERT INTO public.role_capability
+SELECT id, perm FROM public.role
+CROSS JOIN (
+    VALUES ('DNS-SEC:READ'), ('DNS-SEC:DELETE')
+) AS perms(perm)
+WHERE priv_level >= 30
+    ON CONFLICT DO NOTHING;
+
+INSERT INTO public.role_capability (role_id, cap_name)
+SELECT id, perm
+FROM public.role
+CROSS JOIN (
+    VALUES ('DELIVERY-SERVICE-SAFE:UPDATE')
+) AS perms(perm)
+WHERE "priv_level" < 20 AND "priv_level" > 0
+    ON CONFLICT DO NOTHING;
```

Comment:
`INSERT` statements belong in `seeds.sql`. Also, this results in the `admin` role and the `router` role gaining perms they did not previously have (role `1` is `trouter`, role `2` is `admin`):

```diff
--- apache.sql	2023-08-31 14:42:08.747837019 -0600
+++ 7764.sql	2023-08-31 14:42:41.985179003 -0600
@@ -3022,6 +3091,9 @@
 6	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 7	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 8	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
+2	DNS-SEC:READ	A_TIMESTAMP
+2	DNS-SEC:DELETE	A_TIMESTAMP
+1	DELIVERY-SERVICE-SAFE:UPDATE	A_TIMESTAMP
 3	ACME:READ	A_TIMESTAMP
 4	ACME:READ	A_TIMESTAMP
 6	ACME:READ	A_TIMESTAMP
```